org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -467,6 +484,40 @@ private void startZigBeeNetwork() {
         }, 1, TimeUnit.SECONDS);
     }
 
+    private void startReconnectJobIfNotRunning() {
+        if (reconnectPollingTimer != null) {
+            return;
+        }
+
+        reconnectPollingTimer = reconnectPollingScheduler.scheduleWithFixedDelay(new Runnable() {
+            @Override
+            public void run() {
+                ZigBeeTransportState state = networkManager.getNetworkState();
+                if (state == ZigBeeTransportState.ONLINE || state == ZigBeeTransportState.INITIALISING) {
```

Comment:
Shouldn't this also check if we're doing the firmware update, or is it not needed here for some reason?

Reply:
This check is a safeguard in which we hopefully should never run, but in theory could due to a strange race condition.

I am testing here against the status of the ZSS library, not the one from the BridgeHandler. AFAIK there is no such state as "Firmware_Updating", am I missing something?